ResourceStatusSystem/PolicySystem/Actions/test/Test_RSS_PolicySystem_EmailAction.py

```diff
@@ -0,0 +1,84 @@
+''' Test_RSS_PolicySystem_EmailAction
+
+    this is pytest!
+'''
+
+import os
+import json
+from DIRAC.Core.Base.Script import parseCommandLine
+parseCommandLine()
```

Comment:
Not! For unittests, always avoid calling `parseCommandLine`

`parseCommandLine` should be called for integration tests, but unit tests should always be able to run without it.